src/main/java/eu/enmeshed/model/notificationItem/PeerSharedAttributeSucceededNotificationItem.java

```diff
@@ -0,0 +1,15 @@
+package eu.enmeshed.model.notificationItem;
+
+import eu.enmeshed.model.attributes.IdentityAttribute;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.SuperBuilder;
+
+@NoArgsConstructor
+@Getter
+@Setter
+@SuperBuilder
+public class PeerSharedAttributeSucceededNotificationItem extends NotificationItem {
+  private IdentityAttribute successorContent;
```

Comment:
This can also be a RelationshipAttribute.

Reply:
Please do not add another property for that (see other comment). Instead, change the type of `successorContent` from `IdentityAttribute` to `Attribute`. RelationshipAttributes and IdentityAttributes together represent all Attributes.